server/src/graql/reasoner/atom/Atom.java

```diff
@@ -38,6 +39,8 @@
 import grakn.core.graql.reasoner.atom.predicate.VariablePredicate;
 import grakn.core.graql.reasoner.cache.SemanticDifference;
 import grakn.core.graql.reasoner.cache.VariableDefinition;
+import grakn.core.graql.reasoner.query.ReasonerAtomicQuery;
+import grakn.core.graql.reasoner.query.ReasonerQueries;
```

Comment:
are these import needed? I don't see any other code change in this class

Reply:
good catch! they are unused